picker-data-_star_-eu-west-2-policy.json

```diff
@@ -0,0 +1,183 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "cloudformation:List*",
```

Comment:
I feel like this is probably too broad given the `*` resource. We don't do this but you can imagine a scenario where a CloudFormation stack might export some variables that you might not want to be accessible by a third-party.

https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_ListExports.html